launcher/src/backend/NodeConnection.js

```diff
@@ -15,373 +15,481 @@ if(process.env.IS_DEV === 'true'){
+        return reject("Can't list services: " + err);
       }
 
       if (SSHService.checkExecError(serviceJsons)) {
-        return reject('Failed listing services: ' + SSHService.extractExecError(serviceJsons))
+        return reject(
+          "Failed listing services: " +
+            SSHService.extractExecError(serviceJsons)
+        );
       }
 
-      return resolve((serviceJsons.stdout.split(/\n/).slice(0, -1).map(json => {
-        log.debug('parsing json: ', json)
-        return JSON.parse(json)
-      })))
-    })
+      return resolve(
+        serviceJsons.stdout
+          .split(/\n/)
+          .slice(0, -1)
+          .map((json) => {
+            log.debug("parsing json: ", json);
+            return JSON.parse(json);
+          })
+      );
+    });
   }
 
   /**
-     * get details of docker service
-     *
-     * @param serviceId either <docker-container-id> or <'stereum-' + stereum-service-id>
-     */
-  async getServiceDetails (serviceId) {
+   * get details of docker service
+   *
+   * @param serviceId either <docker-container-id> or <'stereum-' + stereum-service-id>
+   */
+  async getServiceDetails(serviceId) {
     return new Promise(async (resolve, reject) => {
-      let serviceJson
+      let serviceJson;
       try {
-        serviceJson = await this.sshService.exec('docker inspect ' + serviceId)
+        serviceJson = await this.sshService.exec("docker inspect " + serviceId);
       } catch (err) {
-        log.error("Can't get service details of '" + serviceId + "': ", err)
-        return reject("Can't get service details of '" + serviceId + "': " + err)
+        log.error("Can't get service details of '" + serviceId + "': ", err);
+        return reject(
+          "Can't get service details of '" + serviceId + "': " + err
+        );
       }
 
       if (SSHService.checkExecError(serviceJson)) {
-        return reject("Failed getting service details of '" + serviceId + "': " + SSHService.extractExecError(serviceJson))
+        return reject(
+          "Failed getting service details of '" +
+            serviceId +
+            "': " +
+            SSHService.extractExecError(serviceJson)
+        );
       }
 
-      return resolve(JSON.parse(serviceJson.stdout))
-    })
+      return resolve(JSON.parse(serviceJson.stdout));
+    });
   }
 
-  async destroyNode () {
-    await this.sshService.exec('docker rm -vf $(docker ps -aq)')
-    await this.sshService.exec('docker rmi -f $(docker images -aq)')
+  async destroyNode() {
+    await this.sshService.exec("docker rm -vf $(docker ps -aq)");
+    await this.sshService.exec("docker rmi -f $(docker images -aq)");
     await this.sshService.exec(`\
                 docker volume prune -f &&\
                 docker system prune -a -f &&\
                 rm -rf ${this.installationDirectory} &&\
-                rm -rf /etc/stereum/services`)
-    return 'Node destroyed'
-  }
 
+                rm -rf /etc/stereum`);
```

Comment:
needs to be `/etc/stereum/services`

Reply:
and could you pls turn of your code refactoring for backend code 😅
it does really weird stuff and makes code hard to read sometimes